localization/ndt_scan_matcher/schema/sub/covariance_covariance_estimation.json

```
@@ -19,8 +19,16 @@
           "type": "array",
           "description": "Offset arrangement in covariance estimation [m]. initial_pose_offset_model_x & initial_pose_offset_model_y must have the same number of elements.",
           "default": [0.5, -0.5, 0.0, 0.0, 0.0, 0.0]
+        },
+        "covariance_estimation_type": {
+          "type": "float",
```

In json schema we can not use "float". Please replace with "number"
```suggestion
          "type": "number",
```

---

Ok! Thanks for pointing that out!